modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -336,13 +336,17 @@ RETURN p
 [#_neo_clientnotification_statement_noapplicableindex]
 === NoApplicableIndex
 
-.When is this notification returned?
-[TIP]
-====
-Given a larger number of nodes, `LOAD CSV` together with a `MATCH` or a `MERGE` clause may speed up the query if you add an index.
-====
+This notification is returned when using `LOAD CSV` with a `MATCH` or a `MERGE` clause that matches a non-indexed label.
+This will most likely not perform well on large data sets.
+If you add an index, `LOAD CSV` together with a `MATCH` or a `MERGE` clause may speed up the query.
```

Comment:
I think this make it sound like it is the `LOAD CSV` rather than the index which speeds up the query. Maybe the word ordering can be changed or the sentence slightly rewritten in some other way to make it clearer?